lib/libutils/ext/include/util.h

```diff
@@ -30,11 +30,20 @@
 	   __typeof__(b) _b = (b); \
 	 _a < _b ? _a : _b; }))
 #else
-#define MAX(a, b)	MAX_RAW(a, b)
-#define MIN(a, b)	MIN_RAW(a, b)
+#define MAX(a, b)	MAX_ASM(a, b)
+#define MIN(a, b)	MIN_ASM(a, b)
 #endif
 #endif
 
+/*
+ * MAX_UNSAFE/MIN_UNSAFE aim at supporting MAX/MIN with
+ * non-typed arguments in some C source file implementation
```

Comment:
The problem with MAX_UNSAFE/MIN_UNSAFE is that the arguments are evaluated more than once.
In assembly mode that's not much of a problem since the arguments there can't have side effects any way.